### ISSUE TITLE
(3.0.65) browse methods containing `%` yields "Can't hold $\x3B1" Dollphin error

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#460]
![image](https://user-images.githubusercontent.com/623951/55685954-8fe4dc00-5910-11e9-8113-11737ee11eac.png)

Here's the dolphin stack:
```
***** Starting Dolphin stack dump at 8:35:01 AM 4/7/2019 *****

************************** Dolphin Virtual Machine Dump Report ***************************

8:35:01 AM, 4/7/2019: Can't hold $\x3B1

*----> VM Context <----*
Process: {1E870210:size 720 words, suspended frame 1E870DF9, priority 6, callbacks 1
last failure 2:nil, FP control 80017, thread 00000000}
Active Method: SessionManager>>logError:
IP: 0230B5D9 (9)
SP: 1E870E54
BP: 1E870E28 (759)
ActiveFrame: {1E870E2C: cf 1E870E0D, sp 1E870E44, bp 1E870E28, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a Error


New Method: VMLibrary>>dump:path:stackDepth:walkbackDepth:
Message Selector: dump:path:stackDepth:walkbackDepth:

*----> Stack Back Trace <----*
{1E870E2C: cf 1E870E0D, sp 1E870E44, bp 1E870E28, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a Error

{1E870E0C: cf 1E870DE9, sp 1E870E20, bp 1E870E04, ip 39, JadeSessionManager>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a Error
	stack temp[0]: '8:35:01 AM 4/7/2019'
	env temp[0]: a FileStream

{1E870DE8: cf 1E870DC9, sp 1E870DFC, bp 1E870DE4, ip 3, JadeSessionManager(SessionManager)>>unhandledException:}
	receiver: a JadeSessionManager
	arg[0]: a Error

{1E870DC8: cf 1E870DA9, sp 1E870DDC, bp 1E870DC4, ip 3, JadeSessionManager(SessionManager)>>onUnhandledError:}
	receiver: a JadeSessionManager
	arg[0]: a Error

{1E870DA8: cf 1E870D8D, sp 1E870DBC, bp 1E870DA8, ip 5, Error>>defaultAction}
	receiver: a Error

{1E870D8C: cf 1E870D59, sp 1E870DA0, bp 1E870D74, ip 55, Error(Exception)>>_propagateFrom:}
	receiver: a Error
	arg[0]: a ExceptionHandler
	stack temp[0]: nil
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: a Process('Main' base 1E870000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02730010)
	stack temp[4]: nil

{1E870D58: cf 1E870D39, sp 1E870D6C, bp 1E870D54, ip 6, Error(Exception)>>_propagate}
	receiver: a Error
	stack temp[0]: nil

{1E870D38: cf 1E870D1D, sp 1E870D4C, bp 1E870D38, ip 13, Error(Exception)>>signal}
	receiver: a Error

{1E870D1C: cf 1E870CFD, sp 1E870D30, bp 1E870D18, ip 5, Error(Exception)>>signal:}
	receiver: a Error
	arg[0]: 'Can't hold $\x3B1'

{1E870CFC: cf 1E870CDD, sp 1E870D10, bp 1E870CF8, ip 5, Error class(Exception class)>>signal:}
	receiver: Error
	arg[0]: 'Can't hold $\x3B1'

{1E870CDC: cf 1E870CBD, sp 1E870CF0, bp 1E870CD8, ip 4, String(Object)>>error:}
	receiver: 'unicodeCharacterSource\0A\\09\^ ($A to: $Z), ($a to: $z), ($0 to: $9), '.-_/*+=|,;?!$&...'
	arg[0]: 'Can't hold $\x3B1'

{1E870CBC: cf 1E870C9D, sp 1E870CD0, bp 1E870CB8, ip 6, String(Object)>>errorCantHold:}
	receiver: 'unicodeCharacterSource\0A\\09\^ ($A to: $Z), ($a to: $z), ($0 to: $9), '.-_/*+=|,;?!$&...'
	arg[0]: $\x3b1

{1E870C9C: cf 1E870C79, sp 1E870CB0, bp 1E870C94, ip 24, String(Object)>>errorAt:put:}
	receiver: 'unicodeCharacterSource\0A\\09\^ ($A to: $Z), ($a to: $z), ($0 to: $9), '.-_/*+=|,;?!$&...'
	arg[0]: 110
	arg[1]: $\x3b1

{1E870C78: cf 1E870C55, sp 1E870C8C, bp 1E870C70, ip 8, String>>at:put:}
	receiver: 'unicodeCharacterSource\0A\\09\^ ($A to: $Z), ($a to: $z), ($0 to: $9), '.-_/*+=|,;?!$&...'
	arg[0]: 110
	arg[1]: $\x3b1

{1E870C54: cf 1E870C35, sp 1E870C68, bp 1E870C50, ip 15, WriteStream>>nextPut:}
	receiver: a WriteStream
	arg[0]: $\x3b1

{1E870C34: cf 1E870C11, sp 1E870C48, bp 1E870C2C, ip 41, [] in STONReader>>parseStringInternal}
	receiver: a STONReader
	arg[0]: a WriteStream

{1E870C10: cf 1E870BF1, sp 1E870C24, bp 1E870C0C, ip 18, STONReader>>stringStreamContents:}
	receiver: a STONReader
	arg[0]: [] @ 27 in STONReader>>parseStringInternal

{1E870BF0: cf 1E870BCD, sp 1E870C04, bp 1E870BE8, ip 47, STONReader>>parseStringInternal}
	receiver: a STONReader
	stack temp[0]: nil
	stack temp[1]: $'

{1E870BCC: cf 1E870BB1, sp 1E870BE0, bp 1E870BCC, ip 2, STONReader>>parseString}
	receiver: a STONReader

{1E870BB0: cf 1E870B91, sp 1E870BC4, bp 1E870BAC, ip 42, STONReader>>parseValue}
	receiver: a STONReader
	stack temp[0]: $'

{1E870B90: cf 1E870B69, sp 1E870BA4, bp 1E870B84, ip 40, STONReader>>parseMapDo:}
	receiver: a STONReader
	arg[0]: [] @ 25 in Object>>fromSton:
	stack temp[0]: source
	stack temp[1]: 36064769

{1E870B68: cf 1E870B41, sp 1E870B7C, bp 1E870B60, ip 33, RowanMethodService(Object)>>fromSton:}
	receiver: a RowanMethodService
	arg[0]: a STONReader
	stack temp[0]: a Array

{1E870B40: cf 1E870B21, sp 1E870B58, bp 1E870B3C, ip 6, RowanMethodService class(Object class)>>fromSton:}
	receiver: RowanMethodService
	arg[0]: a STONReader

{1E870B20: cf 1E870AF9, sp 1E870B34, bp 1E870B14, ip 8, STONReader>>parseObject}
	receiver: a STONReader
	stack temp[0]: RowanMethodService
	stack temp[1]: a STONReference
	stack temp[2]: nil

{1E870AF8: cf 1E870AD5, sp 1E870B0C, bp 1E870AF4, ip 12, STONReader>>parseValue}
	receiver: a STONReader
	stack temp[0]: $R

{1E870AD4: cf 1E870AB1, sp 1E870AEC, bp 1E870ACC, ip 20, STONReader>>parseListDo:}
	receiver: a STONReader
	arg[0]: [] @ 20 in STONReader>>parseList
	stack temp[0]: 100

{1E870AB0: cf 1E870A91, sp 1E870AC4, bp 1E870AAC, ip 25, [] in STONReader>>parseList}
	receiver: a STONReader
	arg[0]: a WriteStream

{1E870A90: cf 1E870A69, sp 1E870AA4, bp 1E870A84, ip 6, Array class(SequenceableCollection class)>>new:streamContents:}
	receiver: Array
	arg[0]: 100
	arg[1]: [] @ 12 in STONReader>>parseList
	stack temp[0]: a WriteStream

{1E870A68: cf 1E870A49, sp 1E870A7C, bp 1E870A64, ip 6, Array class(SequenceableCollection class)>>streamContents:}
	receiver: Array
	arg[0]: [] @ 12 in STONReader>>parseList

{1E870A48: cf 1E870A25, sp 1E870A5C, bp 1E870A40, ip 27, STONReader>>parseList}
	receiver: a STONReader
	stack temp[0]: a STONReference
	stack temp[1]: nil

{1E870A24: cf 1E870A05, sp 1E870A38, bp 1E870A20, ip 28, STONReader>>parseValue}
	receiver: a STONReader
	stack temp[0]: $[

{1E870A04: cf 1E8709E5, sp 1E870A18, bp 1E870A00, ip 4, STONReader>>next}
	receiver: a STONReader
	stack temp[0]: nil

{1E8709E4: cf 1E8709C5, sp 1E8709F8, bp 1E8709E0, ip 5, STON class>>fromStream:}
	receiver: STON
	arg[0]: a ReadStream

{1E8709C4: cf 1E8709A5, sp 1E8709D8, bp 1E8709C0, ip 13, STON class>>fromString:}
	receiver: STON
	arg[0]: '[RowanMethodService{#oop:35004673,#source:'uri\n\t\"Convert my path into a file:...'

{1E8709A4: cf 1E87097D, sp 1E8709B8, bp 1E870998, ip 6, BrowserUpdate>>update:afterStonReplication:}
	receiver: a BrowserUpdate
	arg[0]: a Array
	arg[1]: '[RowanMethodService{#oop:35004673,#source:'uri\n\t\"Convert my path into a file:...'
	stack temp[0]: nil

{1E87097C: cf 1E870951, sp 1E870990, bp 1E87096C, ip 50, BrowserUpdate>>basicIssueCommand:session:}
	receiver: a BrowserUpdate
	arg[0]: a Array
	arg[1]: a GciSession
	stack temp[0]: '[RowanQueryService{#command:#'methodsContaining:',#commandArgs:['%']}]'
	stack temp[1]: nil
	env temp[0]: '[RowanMethodService{#oop:35004673,#source:'uri\n\t\"Convert my path into a file:...'

{1E870950: cf 1E870925, sp 1E870964, bp 1E870948, ip 38, [] in BrowserUpdate>>issueCommand:session:}
	receiver: a BrowserUpdate

{1E870924: cf 1E870905, sp 1E870938, bp 1E870920, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1E870904: cf 1E8708E1, sp 1E870918, bp 1E8708FC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 35 in BrowserUpdate>>issueCommand:session:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1E8708E0: cf 1E8708B9, sp 1E8708F4, bp 1E8708D4, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 35 in BrowserUpdate>>issueCommand:session:
	stack temp[0]: a Cursor
	stack temp[1]: a ExternalHandle(00010007)

{1E8708B8: cf 1E87088D, sp 1E8708CC, bp 1E8708B0, ip 40, [] in BrowserUpdate>>issueCommand:session:}
	receiver: a BrowserUpdate

{1E87088C: cf 1E87086D, sp 1E8708A0, bp 1E870888, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 48 in BrowserUpdate>>issueCommand:session:

{1E87086C: cf 1E870849, sp 1E870880, bp 1E870864, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 24 in BrowserUpdate>>issueCommand:session:
	arg[0]: [] @ 48 in BrowserUpdate>>issueCommand:session:
	stack temp[0]: nil

{1E870848: cf 1E87081D, sp 1E87085C, bp 1E870840, ip 53, [] in BrowserUpdate>>issueCommand:session:}
	receiver: a BrowserUpdate

{1E87081C: cf 1E8707FD, sp 1E870830, bp 1E870818, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 61 in BrowserUpdate>>issueCommand:session:

{1E8707FC: cf 1E8707D9, sp 1E870810, bp 1E8707F4, ip 66, BrowserUpdate>>issueCommand:session:}
	receiver: a BrowserUpdate
	arg[0]: a Array
	arg[1]: a GciSession

{1E8707D8: cf 1E8707B5, sp 1E8707EC, bp 1E8707D0, ip 6, JadeiteProjectsBrowserPresenter class(JadePresenter class)>>issueCommand:session:}
	receiver: JadeiteProjectsBrowserPresenter
	arg[0]: a Array
	arg[1]: a GciSession

{1E8707B4: cf 1E870789, sp 1E8707C8, bp 1E8707A4, ip 14, JadeiteProjectsBrowserPresenter class(JadePresenter class)>>browseMethodsContaining:session:}
	receiver: JadeiteProjectsBrowserPresenter
	arg[0]: '%'
	arg[1]: a GciSession
	stack temp[0]: a RowanQueryService
	stack temp[1]: nil

{1E870788: cf 1E87075D, sp 1E87079C, bp 1E870780, ip 23, [] in JadeiteProjectsBrowserPresenter class(JadePresenter class)>>browseMethodsContaining:}
	receiver: JadeiteProjectsBrowserPresenter

{1E87075C: cf 1E87073D, sp 1E870770, bp 1E870758, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1E87073C: cf 1E870719, sp 1E870750, bp 1E870734, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 20 in JadePresenter class>>browseMethodsContaining:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1E870718: cf 1E8706F1, sp 1E87072C, bp 1E87070C, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 20 in JadePresenter class>>browseMethodsContaining:
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(00010003)

{1E8706F0: cf 1E8706CD, sp 1E870704, bp 1E8706E8, ip 25, JadeiteProjectsBrowserPresenter class(JadePresenter class)>>browseMethodsContaining:}
	receiver: JadeiteProjectsBrowserPresenter
	arg[0]: a GciSession
	stack temp[0]: '%'

{1E8706CC: cf 1E8706B1, sp 1E8706E0, bp 1E8706CC, ip 5, JadeiteProjectsBrowserPresenter(JadePresenter)>>browseMethodsContaining}
	receiver: a JadeiteProjectsBrowserPresenter

{1E8706B0: cf 1E870691, sp 1E8706C4, bp 1E8706AC, ip 4, Symbol>>forwardTo:}
	receiver: browseMethodsContaining
	arg[0]: a JadeiteProjectsBrowserPresenter

{1E870690: cf 1E870671, sp 1E8706A4, bp 1E87068C, ip 4, CommandDescription>>performAgainst:}
	receiver: a CommandDescription
	arg[0]: a JadeiteProjectsBrowserPresenter

{1E870670: cf 1E87064D, sp 1E870684, bp 1E870670, ip 14, [] in Command>>value}
	receiver: a Command

{1E87064C: cf 1E87062D, sp 1E870660, bp 1E870648, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 17 in Command>>value

{1E87062C: cf 1E870609, sp 1E870640, bp 1E870624, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in Command>>value
	arg[0]: [] @ 17 in Command>>value
	stack temp[0]: nil

{1E870608: cf 1E8705ED, sp 1E87061C, bp 1E870608, ip 22, Command>>value}
	receiver: a Command

{1E8705EC: cf 1E8705CD, sp 1E870600, bp 1E8705E8, ip 30, ShellView>>performCommand:}
	receiver: a ShellView
	arg[0]: a Command

{1E8705CC: cf 1E8705AD, sp 1E8705E0, bp 1E8705C8, ip 4, JadeiteProjectsSystemBrowser(Shell)>>performCommand:}
	receiver: a JadeiteProjectsSystemBrowser
	arg[0]: a Command

{1E8705AC: cf 1E870585, sp 1E8705C0, bp 1E8705A0, ip 10, CommandQuery>>perform}
	receiver: a CommandQuery
	stack temp[0]: a Command
	stack temp[1]: nil
	stack temp[2]: a JadeiteProjectsSystemBrowser

{1E870584: cf 1E870561, sp 1E870598, bp 1E87057C, ip 8, DelegatingCommandPolicy(CommandPolicy)>>route:}
	receiver: a DelegatingCommandPolicy
	arg[0]: a CommandDescription
	stack temp[0]: a CommandQuery

{1E870560: cf 1E870539, sp 1E870574, bp 1E87055C, ip 15, [] in ShellView(View)>>onCommand:}
	receiver: a ShellView

{1E870538: cf 1E870519, sp 1E87054C, bp 1E870534, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1E870518: cf 1E8704F5, sp 1E87052C, bp 1E870510, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in View>>onCommand:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1E8704F4: cf 1E8704CD, sp 1E870508, bp 1E8704E8, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 11 in View>>onCommand:
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(00010003)

{1E8704CC: cf 1E8704A9, sp 1E8704E0, bp 1E8704C4, ip 17, ShellView(View)>>onCommand:}
	receiver: a ShellView
	arg[0]: a CommandDescription
	stack temp[0]: nil

{1E8704A8: cf 1E870475, sp 1E8704BC, bp 1E870490, ip 19, ShellView(View)>>wmCommand:wParam:lParam:}
	receiver: a ShellView
	arg[0]: 273
	arg[1]: 6164
	arg[2]: 0
	stack temp[0]: 6164
	stack temp[1]: a CommandDescription
	stack temp[2]: nil

{1E870474: cf 1E870445, sp 1E870488, bp 1E870460, ip 23, ShellView(View)>>dispatchMessage:wParam:lParam:}
	receiver: a ShellView
	arg[0]: 273
	arg[1]: 6164
	arg[2]: 0
	stack temp[0]: nil
	stack temp[1]: wmCommand:wParam:lParam:

{1E870444: cf 1E870411, sp 1E870458, bp 1E870434, ip 69, [] in InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: nil

{1E870410: cf 1E8703E9, sp 1E870424, bp 1E87040C, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 79 in InputState>>wndProc:message:wParam:lParam:cookie:

{1E8703E8: cf 1E8703AD, sp 1E870404, bp 1E8703D0, ip 84, InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: a InputState
	arg[0]: 590476
	arg[1]: 273
	arg[2]: 6164
	arg[3]: 0
	arg[4]: 851284
	stack temp[0]: a ShellView

{1E8703AC: cf 1E87038D, sp 1E8703C8, bp 1E8703A8, ip 28, InputState>>pumpMessage:}
	receiver: a InputState
	arg[0]: a MSG

{1E87038C: cf 1E870361, sp 1E8703A0, bp 1E87037C, ip 18, InputState>>loopWhile:}
	receiver: a InputState
	arg[0]: [] @ 9 in InputState>>mainLoop
	stack temp[0]: a MSG
	stack temp[1]: true
	stack temp[2]: nil

{1E870360: cf 1E870345, sp 1E870374, bp 1E870360, ip 15, InputState>>mainLoop}
	receiver: a InputState

{1E870344: cf 1E870329, sp 1E870358, bp 1E870344, ip 14, [] in InputState>>forkMain}
	receiver: a InputState

{1E870328: cf 1E87030D, sp 1E87033C, bp 1E870328, ip 13, ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry}
	receiver: a ExceptionHandler

{1E87030C: cf 1E8702E5, sp 1E870320, bp 1E870308, ip 22, [] in ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler

{1E8702E4: cf 1E8702C5, sp 1E8702F8, bp 1E8702E0, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:

{1E8702C4: cf 1E8702A1, sp 1E8702D8, bp 1E8702BC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 17 in ExceptionHandlerAbstract>>try:
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:
	stack temp[0]: nil

{1E8702A0: cf 1E870279, sp 1E8702B4, bp 1E870294, ip 44, ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler
	arg[0]: [] @ 10 in InputState>>forkMain
	stack temp[0]: nil
	stack temp[1]: a Process('Main' base 1E870000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02730010)
	env temp[0]: nil

{1E870278: cf 1E870255, sp 1E87028C, bp 1E870270, ip 7, BlockClosure>>on:do:}
	receiver: [] @ 10 in InputState>>forkMain
	arg[0]: ProcessTermination
	arg[1]: [] @ 13 in BlockClosure>>newProcess

{1E870254: cf 00000001, sp 1E870268, bp 1E870254, ip 17, [] in BlockClosure>>newProcess}
	receiver: [] @ 10 in InputState>>forkMain

<Bottom of stack>

***** End of dump *****

***** Ending Dolphin stack dump started at 8:35:01 AM 4/7/2019 *****
```

**Comment by ericwinger** [comment]
The culprit is this method which contains unicode characters. Need to official support unicode in Jadeite (Dolphin). 

```
topaz 2> look ZnCharacterEncoderTests class >> unicodeCharacterSource
    category: 'accessing'
classmethod: ZnCharacterEncoderTests
unicodeCharacterSource
        ^ ($A to: $Z), ($a to: $z), ($0 to: $9), '.-_/*+=|,;?!$&<>^%#', '         ', 'éèçüäßñα', '€∏'
%
```

**Comment by ericwinger** [comment]
See issued #307 and #453

**Comment by ericwinger** [comment]
One stopgap possibility is to check each method service's source before returning to the client via STON. If it contains unicode, replace the source with an uncompilable ascii string. That string could include the UTF8 string for the original source with an explanation of what is presented.

**Comment by ericwinger** [comment]
@dalehenrich @LisaAlmarode  what do you think about this presentation of an "undisplayed" source below? 

Instead of the UTF8 string, I look for characters > 255 and present those in their hex form. Note that the source is not  intended to be compilable. 

![image](https://user-images.githubusercontent.com/36212516/56323114-b92b1680-611f-11e9-94ba-c95efa552170.png)

**Comment by dalehenrich** [comment]
that would seem to be okay as long as it is obvious that that translated sequence is really a replacement for an undisplayable character ... maybe a purple background is needed, since you should be disallowed from saving the method ...

**Comment by ericwinger** [comment]
I added an all caps warning and made the background the same color as a compiler error. Thoughts? 

![image](https://user-images.githubusercontent.com/36212516/56326501-24c6b100-612b-11e9-8d79-3dea9dddd81c.png)

**Comment by dalehenrich** [comment]
works for me:)

**Comment by ericwinger** [comment]
The above pane should show up if a unicode character is encountered as of 3.0.67 https://github.com/GemTalk/Jadeite/commit/079848be53c30ab5ae866e4b2a7b41075280426d

**Comment by LisaAlmarode** [comment]
It works pretty well, but you can't see when the replacement character ends so it's not reliably readable.  Some kind of separator (like () {} [] ) encasing the subbed hex would remove ambiguity. 

For example, in topaz
classmethod Array
unclearExotic
^'Ȋc€'
%
commit

and in Oscar:

![image](https://user-images.githubusercontent.com/13142166/62389240-266a8800-b514-11e9-9908-8610959e89fe.png)

**Comment by LisaAlmarode** [comment]
The issue as described is fixed as of 3.2.12.  There are still issues with unicode, but these should get their own issues.
